terraform.tfstate

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
I'm assuming that terraform.tfstate should not be committed? Likely need a gitignore, something like https://github.com/github/gitignore/blob/master/Terraform.gitignore?